src/Alarm.cpp

```diff
@@ -14,7 +14,7 @@
 #include "ComponentTester.h"
 
 Alarm::Alarm(){
-  Logger::Log("Alarm has been created");
+  Serial.write(14016); //14016 = Log, Info Alarm Created
```

Comment:
This is a little nit-picky, but this doesn't quite match the comment convention defined in the [Developer Code Comment Convention Documentation](https://github.com/Lastrellik/Calico-Home-Security/blob/4f540c026c61163b1bf5dc57533bb2c4d0915a3d/docs/development/logging.md#developer-code-comment-convention). It should be `// ####### = X, Y, Z`. You've got it as `// ####### = X, Y Z` - note the missing comma between `Y` and `Z`.

There are several of these scattered throughout this Pull Request.